alexandria/core/serializers.py

```diff
@@ -363,3 +359,23 @@ class Meta:
             "name",
         )
         read_only_fields = fields
+
+
+class CopyCategorySerializer(Serializer):
+    data = serializers.DictField()
+
+    def validate(self, data):
+        validated_data = super().validate(data)
+        category_id = validated_data.get("data", {}).get("category")
+
+        if category_id:
+            try:
+                category = models.Category.objects.get(pk=category_id)
+            except models.Category.DoesNotExist:
+                raise serializers.ValidationError("Category not found")
+        else:
+            category = None
+
+        validated_data["category"] = category
+
+        return validated_data
```

Comment:
Instead of having `data` as a `DictField`, I'd put the category as it's own `ResourceRelatedField`. Build a serializer for the whole request, not just for the category part.

Reply:
Took me some time to figure this one out and changed the request format a bit to work with the `ResourceRelatedField`. Can you verify if the new version looks good? If so I will update the request in the `ember-alexandria` MR (https://github.com/projectcaluma/ember-alexandria/pull/1079) as well to reflect this change.

Reply:
I wonder what we'd need to do to have the request be in a proper [JSONAPI](https://jsonapi.org/) format. Fine for me as it is now, but still I think I'd like it to be even more consistent :-)